build/_build.csproj

```diff
@@ -13,13 +13,16 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="ILRepack" Version="2.0.18" />
-    <PackageReference Include="Nuke.Common" Version="7.0.6" />
+    <PackageReference Include="ILRepack" Version="2.0.18">
```

Comment:
Story: I upgraded ILRepack to the latest 2.0.35, however the test which asserts things get converted to `internal` failed; TinyTypes were being exposed as public, so I rolled it back.

The PrivateAssets thing was added by rider as part of the package upgrade/downgrade, it doesn't have any effect here anyway